src/custom-element.js

```diff
@@ -904,6 +904,9 @@ function createBaseCustomElementClass(win) {
           setStyle(this, 'marginLeft', opt_newMargins.left, 'px');
         }
       }
+      if (this.isAwaitingSize_()) {
```

Comment:
Would rather not put logic like that here. There are a number of scenarios e.g.:
- margins changed but height not
- width changed but height not
- actually wanting zero height and the element to be in the flow

I feel like we have to second guess all permutations of what a caller might legitimately want to do and seems like unnecessary complexity.